images/router/haproxy/conf/error-page-404.http

```diff
@@ -0,0 +1,148 @@
+HTTP/1.0 404 Error

```

Comment:
Is this new file identical to `images/router/haproxy/conf/error_code_pages/error-page-404.http`? We don't need both of them, right?

Reply:
Yeah I had copied both files from the existing ones. Actually we need to have a backup somewhere so have not touched them.